modules/vending/vendors.php

```diff
@@ -0,0 +1,53 @@
+<?php
+if (!defined('FLUX_ROOT')) exit;
+
+$title = 'Vendors';
+
+$vendor = trim($params->get('vendor'));
+$shopname = trim($params->get('title'));
+$map = trim($params->get('map'));
+
+$sql_criteria = "";
+$sql_params = array();
+if ($vendor) {
```

Comment:
```suggestion
if (preg_match('/^[\w\'\- ]+$/', $vendor)) {
```
More allowed char name on server?